cypress/integration/core-opensearch-dashboards/vanilla-opensearch-dashboards/datasource-management-plugin/2_datasource_table.spec.js

```diff
@@ -0,0 +1,436 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { MiscUtils } from '@opensearch-dashboards-test/opensearch-dashboards-test-library';
+
+const miscUtils = new MiscUtils(cy);
+
+describe('Datasource Management: Table', () => {
+  beforeEach(() => {
+    // Visit Data Sources OSD
+    miscUtils.visitPage('app/management/opensearch-dashboards/dataSources');
```

Comment:
This page will not load because we need config to enable data source plugin. Let's figure out how to override the config.

Reply:
Added the feature flag and if condition